Detector/DetFCChhHCalTile/compact/FCChh_HCalBarrel_TileCal.xml

```diff
@@ -2,30 +2,34 @@
 <lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
   xmlns:xs="http://www.w3.org/2001/XMLSchema"
   xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
-
   <info name="FCChh_HCalBarrel_TileCal"
-    title="Barrel Calorimeters"
-    author="C. Solans, J. Lingemann"
+    title="Barrel Calorimeter"
+    author="C. Neubueser"
     url="no"
     status="development"
     version="1.0">
-    <comment>HCal layout based on ATLAS HCal</comment>
+    <comment>HCal layout based on ATLAS HCal, with realistic longitudinal segmentation and steel support</comment>
   </info>
-  
   <define>
-    <!-- 20cm for support is reduced to 18cm, 1cm is needed to place max number of trapazoids and 1cm needed to compensate for faceplate-->
-    <constant name="BarHCal_rmaxWithBuffer" value="BarHCal_rmax-180*mm"/>
+    <!-- 21.2cm for support, 1cm is needed to place max number of trapazoids and 1cm needed to compensate for faceplate-->
+    <constant name="BarHCal_rmaxWithBuffer" value="BarHCal_rmax-212*mm"/>
     <!-- dimensions in R / rho of actual detector modules (face-plate is placed inside)-->
     <!-- tile dimensions -->
     <constant name="BarHCal_n_phi_modules" value="128"/>
-    <constant name="BarHCal_module_drho" value="100*mm"/>
+    <constant name="BarHCal_n_layers_10" value="2"/>
+    <constant name="BarHCal_n_layers_15" value="4"/>
+    <constant name="BarHCal_n_layers_25" value="4"/>
+    <constant name="BarHCal_module_drho_10" value="100*mm"/>
+    <constant name="BarHCal_module_drho_15" value="150*mm"/>
+    <constant name="BarHCal_module_drho_25" value="250*mm"/>
```

Comment:
Wouldn't names containing `layer1`, `layer2`, `layer3` make more sense here than the thickness in cm?